ui/src/literature/components/LiteratureItem.jsx

```diff
@@ -61,25 +67,27 @@ class LiteratureItem extends Component {
           </Fragment>
         }
       >
-        <Link className="f4" to={`${LITERATURE}/${recordId}`}>
+        <Link className="f5" to={`${LITERATURE}/${recordId}`}>
           <LiteratureTitle title={title} />
         </Link>
-        <div className="mt2">
-          <div>
-            <AuthorsAndCollaborations
-              authorCount={authorCount}
-              authors={authors}
-              collaborations={collaborations}
-              collaborationsWithSuffix={collaborationsWithSuffix}
-            />
-          </div>
-          <LiteratureDate date={date} />
+        <div className="mt1">
+          <AuthorsAndCollaborations
+            authorCount={authorCount}
+            authors={authors}
+            collaborations={collaborations}
+            collaborationsWithSuffix={collaborationsWithSuffix}
+          />
+          {' ('}
+          <LiteratureDate wrapperClassName="wrap-in-parentheses" date={date} />
```

Comment:
`wrapperClassName` is leftover, also you might wanna check if `date` is present before putting parenthesis.